dbflow-processor/src/main/java/com/raizlabs/android/dbflow/processor/definition/method/provider/QueryMethod.java

```diff
@@ -53,8 +53,8 @@ public MethodSpec getMethodSpec() {
                                             .add("$L.PROPERTY_CONVERTER", tableDefinition.getPropertyClassName())
                                             .build());
                     ProviderMethodUtils.appendTableName(codeBuilder,
-                            manager.getDatabaseName(contentProviderDefinition.databaseName), tableEndpointDefinition.tableName);
-                    codeBuilder.add(".where()");
+                        manager.getDatabaseName(contentProviderDefinition.databaseName), tableEndpointDefinition.tableName);
+                    codeBuilder.add(".where(toConditions(selection, selectionArgs))");
```

Comment:
+1 -- content providers is broken because of this.  anyway to merge sooner than later?  we've reverted to dbflow2 because of it